src/pages/admin/create-blog.tsx

```diff
@@ -24,18 +24,9 @@ import './create-blog.scss';
 
 Amplify.configure(awsmobile);
 const federated = {
-    google_client_id: '',
-    facebook_app_id: '579712102531269',
-    amazon_client_id: ''
+  facebookAppId: '579712102531269'
 };
 
-const Index = () => (
-    <div>
-        <Authenticator federated={federated} hide={[Greetings, SignOut]}>
-            <AuthIndexApp></AuthIndexApp>
-        </Authenticator>
-    </div>
-)
 interface Props {
```

Comment:
@nikklassen I could use some advise here. You'll see that `props.authState` is now unused in all `admin/*.tsx` files. How would you pass empty props to the component? Is there a convention for doing something like this in Typescript?

Reply:
Not quite sure if this is what you're going for, but I would say that default (empty), props are `{}`. So if a component has no props you would change it to
```typescript
class Index extends React.Component<{}, State> {
```
If you take a look at the definition of Component you can see that `{}` is what's used as the default type for `Props` and `State` if you don't specify them.

Reply:
Hmmm. I had that initially, but eslint said "don't use {} as a type".

What do you think about:

```typescript
class Index extends React.Component<unknown, State> {
    constructor(props = {}) {
        super(props)
        this.state = { ... }
    }
}
```

Reply:
Hmm, maybe using `{}` is a bad habit. I'll check with my friend who uses React more.

Reply:
There are a few places in JC as well that could use a solution to this...I think the one I had got rid of the ESLint error but then Safari echo's a huge warning to the log which is also not ideal...

Reply:
My friend thinks that this is the best way to do it, because it really is the right type. I think I'd just use the one-line eslint disable for cases. More confirmation: https://stackoverflow.com/questions/50690271/typescript-react-state-without-props

Reply:
Ok I’ll to with `{}` then. Thanks Nik.